syntax = "proto3";

package teos;

import "appointment.proto";
import "google/protobuf/empty.proto";

service RPC_API {
  rpc get_all_appointments(google.protobuf.Empty) returns (GetAllAppointmentsResponse) {}
}